UpdateStatus.sh

```diff
@@ -1,30 +1,76 @@
 #!/bin/bash
 
+################################
+##---- CHANGE ME
+user_email="jalmond@cern.ch"
+#website_dir="/eos/user/" 
+website_dir="/afs/cern.ch/user/" 
+################################
+
 firstletter=`echo $USER | head -c 1`
 mkdir -p $SKFlatTag
-mkdir -p /eos/user/$firstletter/$USER/www/SKFlat/ProductionStatus/$SKFlatTag/
+mkdir -p $website_dir$firstletter/$USER/www/SKFlat/ProductionStatus/$SKFlatTag/
```

Comment:
```suggestion
mkdir -p $website_dir/$firstletter/$USER/www/SKFlat/ProductionStatus/$SKFlatTag/
```
Can we have it like this? Same, but to be clearer...